backend/app/main.py

```diff
@@ -22,6 +22,7 @@
 origins = [
     "http://localhost",
     "http://localhost:8001",
+    f"https://{FRONTEND_DOMAIN}"
```

Comment:
FWIW, this doesn't do anything at the moment since `ALLOW_ALL_ORIGINS = True` above. Since we're serving public data from the API and don't have any sensitive data on the site, I think it's not that big a deal to just let anyone perform queries against the API.